src/App.js

```diff
@@ -1,25 +1,29 @@
-import logo from './logo.svg';
 import './App.css';
+import Counter from "./components/Counter";
+import SearchForm from "./components/SearchForm";
+import GenreSelect from "./components/GenreSelect";
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+export default function App() {
+    function handleSearch(searchInput) {
+        console.log("Input from the search bar: ", searchInput);
+    }
+
+    const genreNames = ['All', 'Documentary', 'Comedy', 'Horror', 'Crime'];
+
+    const selectedGenre = 'Documentary';
 
-export default App;
+    function handleGenreSelect(genre) {
+        console.log("Selected genre: ", genre)
+    }
+
+    return (
+        <div className="App">
+            <Counter initialCount={3}/>
+            <SearchForm initialSearch={"What do you want to watch?"} onSearch={handleSearch}/>
+            <GenreSelect genreNames={genreNames}
+                         selectedGenre={selectedGenre}
+                         onSelect={handleGenreSelect}>
+            </GenreSelect>
```

Comment:
You didn't pass anything between <GendreSelect></...>. You can use short annotation.